salt/modules/pkgng.py

```diff
@@ -475,7 +475,7 @@ def update_package_site(new_url):
     return True
 
 
-def stats(local=False, remote=False, jail=None, chroot=None, root=None):
+def stats(local=False, remote=False, bytes=False, jail=None, chroot=None, root=None):
```

Comment:
Hmm, `bytes` is a python built-in function....
I wonder if this works of if we'll mess something up....

Reply:
`unit="bytes"`?

Reply:
What units are accepted?

Reply:
> What units are accepted?

bytes or human readable format according to IEC 80000-3 standards: `Ki/Mi/Gi/Ti/Pi/Ei`

Reply:
So you can pass all of those values?

Reply:
No, you might pass only bytes, by default it's human readable format

Reply:
Well, if the test suite didn't choke on it, I guess its fine being bytes